src/entities/trades/curve/abi/abi.ts

```diff
@@ -50,6 +50,13 @@ export const CURVE_ETHXERC20_ABI: ContractInterface = [
   poolMethods['payable']['exchange(int128,int128,uint256,uint256)'],
 ]
 
+export const CURVE_TRICRYPTO_ABI: ContractInterface = [
+  poolMethods['view']['fee'],
+  poolMethods['view']['get_dy(uint256,uint256,uint256)'],
+  poolMethods['payable']['exchange(uint256,uint256,uint256,uint256)'],
+  poolMethods['payable']['exchange(uint256,uint256,uint256,uint256,bool)'],
```

Comment:
Related to the comment above about ETH. payable modifier allows the contract to accept the native token

Reply:
I'm only responding for this particular case.
This ABI is only used in 1 place and it's ABI implements these payable methods.